packages/marketplace/src/components/pages/admin-stats/admin-stats.tsx

```diff
@@ -17,20 +16,32 @@ import styles from '@/styles/pages/admin-stats.scss?mod'
 import { AdminStatsRequestParams, adminStatsRequestData } from '@/actions/admin-stats'
 import { Line } from 'react-chartjs-2'
 import { getChartData, getChartConfig, getRangeName } from '@/utils/admin-stats'
+import { selectAdminStats } from '@/selector/admin'
 
 export type Area = 'APPS' | 'DEVELOPERS' | 'INSTALLATIONS'
 export type Range = 'WEEK' | 'MONTH' | 'ALL'
 
-export type AdminStatsProps = DispatchProps & StateProps
+export const handleLoadStats = (dispatch: Dispatch) => (params: AdminStatsRequestParams) => {
+  dispatch(adminStatsRequestData(params))
+}
+
+export const AdminStats: React.FC = () => {
+  const dispatch = useDispatch()
+  const adminStats = useSelector(selectAdminStats)
+
+  const {
```

Comment:
Destructuring like this will cause app crash if adminStats undefined.

Reply:
<img width="366" alt="Screen Shot 2020-06-01 at 10 55 29 AM" src="https://user-images.githubusercontent.com/30467048/83374651-72b57b80-a3f6-11ea-8945-78770443d833.png">

because adminStats had default state, and it is required by the AdminStatsState interface. So I think it cannot be undefined.